vulnerable_people_form/form_pages/shared/form_utils.py

```diff
@@ -58,5 +58,13 @@ def format_postcode(postcode):
     return None
 
 
+def postcode_with_spaces(postcode):
+    if postcode is None or len(postcode) < 3:
+        return ''
```

Comment:
Should this return an empty string? If it is mandatory from where it is called then it would just blow up on submission or further down the line

Reply:
not blowing up is deliberate since this is intended only to be display logic but it would make sense to return something - I'll return `postcode` in the event of the input not being parsable as a postcode